test/test_util.py

```diff
@@ -48,6 +48,7 @@
 # numbers used for timeouts
```

Comment:
I'll just revert this whole file and open a separate PR for the dictionary issue. If flaking the unit tests is something that is desired I will put all the work into one PR. I find it difficult to work on the tests and find actual deficiencies when my IDE is lit up like a Christmas tree with linting errors.